src/Periodic/LargePeriodicPlainTextFormatter.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace CultuurNet\CalendarSummaryV3\Periodic;
+
+use DateTime;
+use IntlDateFormatter;
+
+/**
+ * Provide a large plain text formatter for periodic calendar type.
+ * @package CultuurNet\CalendarSummaryV3\Periodic
+ */
+class LargePeriodicPlainTextFormatter implements PeriodicFormatterInterface
+{
+
+    /**
+     * Translate the day to short Dutch format.
+     */
+    protected $mappingDays = array(
+        'monday' => 'ma',
+        'tuesday' => 'di',
+        'wednesday' => 'wo',
+        'thursday' => 'do',
+        'friday' => 'vr',
+        'saturday' => 'za',
+        'sunday' => 'zo',
+    );
+
+    /**
+     * Return formatted period string.
+     *
+     * @param \CultuurNet\SearchV3\ValueObjects\Offer|\CultuurNet\SearchV3\ValueObjects\Place $place
+     * @return string
+     */
+    public function format($place)
+    {
+        $output = $this->generateDates($place->getStartDate(), $place->getEndDate());
+
+        if ($place->getOpeningHours()) {
+            $output .= PHP_EOL . $this->generateWeekScheme($place->getOpeningHours());
+        }
+
+        return $output;
+    }
+
+    /**
+     * @param $time
+     * @return string
+     */
+    protected function getFormattedTime($time)
+    {
+        $formattedShortTime = ltrim($time, '0');
+        if ($formattedShortTime == ':00') {
+            $formattedShortTime = '0:00';
+        }
+        return $formattedShortTime;
+    }
+
+    /**
+     * @param DateTime $dateFrom
+     * @param DateTime $dateTo
+     * @return string
+     */
+    protected function generateDates(DateTime $dateFrom, DateTime $dateTo)
+    {
+        $fmt = new IntlDateFormatter(
+            'nl_BE',
+            IntlDateFormatter::FULL,
+            IntlDateFormatter::FULL,
+            date_default_timezone_get(),
+            IntlDateFormatter::GREGORIAN,
+            'd MMMM yyyy'
+        );
+
+        $dateFromTimestamp = $dateFrom->getTimestamp();
```

Comment:
Same remark as for large period. Give the datetime object to intl, instead of timestamp